specifyweb/frontend/js_src/lib/localization/query.ts

```diff
@@ -650,28 +650,8 @@ export const queryText = createDictionary({
   importNoReadPermission: {
     'en-us':
       'The query you imported contains tables you do not have read access to:',
-    'es-es': `
```

Comment:
Can you try running this:
`git checkout production; git fetch; git reset --hard origin/production`

Reply:
This branch doesn't even open for me:

![Screenshot 2023-02-28 at 09 20 20](https://user-images.githubusercontent.com/40512816/221897638-90eba3c9-0ade-484b-a996-9da155981f40.png)

Reply:
Fixed by removing the merge commits